pkg/scheduler/algorithmprovider/defaults/defaults_test.go

```diff
@@ -85,3 +87,13 @@ func TestDefaultPredicates(t *testing.T) {
 		t.Errorf("expected %v got %v", expected, result)
 	}
 }
+
+func TestApplyFeatureGates_EnabledResourceLimitsPriorityFunction(t *testing.T) {
+	utilfeature.DefaultFeatureGate.SetFromMap(map[string]bool{string(features.ResourceLimitsPriorityFunction): true})
+	ApplyFeatureGates()
```

Comment:
What are we going to check in this test? It seems always passed :)

Reply:
+1 - this test seems useless to me.